pymatgen/io/lobster/lobsterenv.py

```diff
@@ -214,8 +213,8 @@ def molecules_allowed(self):
 
     def get_anion_types(self):
```

Comment:
Maybe this could be a property?
```suggestion
    @property
    def anion_types(self):
```

Reply:
Yes. It will just break some code in lobsterpy.

Reply:
I have now implemented this but the `get_anion_types` method will still exit until we have made the changes in lobsterpy.